wcivf/apps/core/context_processors.py

```diff
@@ -2,9 +2,15 @@
 
 from .forms import PostcodeLookupForm
 
+# Remove this once we add all_settings to settings.CONTEXT_PROCESSORS
 def canonical_url(request):
     return {'CANONICAL_URL': settings.CANONICAL_URL}
 
+
+def all_settings(request):
+    return dict(settings={x: getattr(settings, x) for x in dir(settings)})
```

Comment:
I mean, I know where you're coming from, but I don't think you really need to because Django will blow up with a `TemplateSyntaxError` if you ever try to access anything that starts with an underscore... 

The only reason I was doing this is that I'm used to it from e.g. https://github.com/openhealthcare/opal/blob/v0.8.2/opal/context_processors.py#L20 which has not caused us a problem in the 3 years since we've been putting it in every application : )